benchmarks/stanza_benchmark.py

```diff
@@ -0,0 +1,29 @@
+import stanza
+# stanza.download('en')
+nlp = stanza.Pipeline('en')
```

Comment:
I think this might be a little unfair on stanza, as you could just run their tokenizer/segementer rather than the full pipeline, e.g:
`nlp = stanza.Pipeline(lang='en', processors='tokenize')`

https://stanfordnlp.github.io/stanza/tokenize.html

Reply:
Resolved